src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -112,22 +112,25 @@ FUZZ_TARGET(coinselection)
     std::vector<OutputGroup> group_all;
     GroupCoins(fuzzed_data_provider, utxo_pool, coin_params, /*positive_only=*/false, group_all);
 
+    // Calculate upperbound of given input weight, allow all to be selected
+    int total_input_weight{0};
```

Comment:
It would be better to make `max_input_weight` a fuzzed value.

Reply:
If you do, please also add an `assert` that checks whether the fuzzed value is greater than or equal to the result’s weight.